configuration/legacy/contact-summary.md

```diff
@@ -101,6 +102,35 @@ if (contact.type === 'person') {
   ];
 }
 
+if (targets) {
+  var activityCard = {
+    label: 'Activity this month',
+    fields: [{
+      label: 'Targets data last updated',
+      value: targets.updated_date,
+      translate: false,
+      filter: 'relativeDate',
+      width: 12,
+    }]
+  };
+
+  targets.targets.forEach(target => {
```

Comment:
would this be a cleaner interface to use `targetDoc.targets` instead of `targets.targets`?

Reply:
Agreed. Let's make this change.